docs/src/examples/components/Chat/Content/ChatExampleActions.shorthand.steps.ts

```diff
@@ -0,0 +1,17 @@
+import { ChatMessage } from '@fluentui/react'
+
+const selectors = {
+  message: `.${ChatMessage.className}`,
+}
+
+const config: ScreenerTestsConfig = {
+  themes: ['teams', 'teamsDark', 'teamsHighContrast'],
+  steps: [
+    builder => builder.hover(selectors.message).snapshot('Hovers the first message'),
+    builder => builder.click(selectors.message).snapshot('Focus the first message via mouse click'),
+    (builder, keys) =>
+      builder.keys('body', keys.tab).snapshot('Focuses the first message via keyboard'),
```

Comment:
nit: technically, this does not select the first message

Reply:
Good catch, renamed this step